processing/src/main/java/org/apache/druid/query/aggregation/AggregatorFactory.java

```diff
@@ -120,9 +144,15 @@ public AggregatorFactory getMergingFactory(AggregatorFactory other) throws Aggre
   }
 
   /**
-   * Gets a list of all columns that this AggregatorFactory will scan
+   * Used by GroupByStrategyV1 when running nested groupBys, to "transfer" values from inner query results to an
+   * incremental index that the outer query will run on. This method is a bit goofy, and should probably not be used
```

Comment:
Even after looking up "goofy" in a dictionary, I don't have a firm sense of what does this word mean. Maybe better to a less slang word.

Reply:
It's meant to mean sort of "silly" or "ideally shouldn't need to exist, but has to because of how something else is designed". The idea I was trying to communicate is that GroupByStrategyV1 is doing something non-ideal here and that requires that this method exist.

I changed the wording to hopefully make this more clear.